app/src/components/Verto/LeftMenu.vue

```diff
@@ -73,6 +80,9 @@ export default {
     }
   },
   methods: {
+    goToLink () {
```

Comment:
Is this how it's done in other places in verto?   I think this may not work on iOS or Android